commondata_projections_L0/LEP3_240_H_HADR.yaml

```
@@ -0,0 +1,47 @@
+dataset_name: LEP3_240_H_HADR
+doi: https://doi.org/10.17181/3jjdh-6fz979
```

Super small detail I noticed by chance: the last 9 in the DOI is incorrect.
```suggestion
doi: https://doi.org/10.17181/3jjdh-6fz97
```